### ISSUE TITLE
Where to get the source code for the demo app?

### ISSUE THREAD

**Issue by mapperkids** [joni2back/angular-filemanager#55]
Hi,

Can I get the source code for the demo app? http://zendelsolutions.com/zendel/projects/angular-filemanager/

Thanks,

**Comment by joni2back** [comment]
Hello, I dont understand your question, can you explain it with more details?
Thanks

**Comment by joni2back** [comment]
You can use the JAVA or the PHP connector

https://github.com/joni2back/angular-filemanager/tree/master/bridges

**Comment by starspn** [comment]
Hello, 

I did all this all this steps >>

Use in your existing project

1) Install and use bower install --save angular-filemanager

2) Include the dependencies in your project


<script src="bower_components/angular/angular.min.js"></script>
<script src="bower_components/angular-translate/angular-translate.min.js"></script>
<script src="bower_components/angular-cookies/angular-cookies.min.js"></script>
<script src="bower_components/jquery/dist/jquery.min.js"></script>
<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
<link rel="stylesheet" href="bower_components/bootswatch/paper/bootstrap.min.css" />

<script src="dist/angular-filemanager.min.js"></script>
<link rel="stylesheet" href="dist/angular-filemanager.css">
3) Use the angular directive in your HTML

<angular-filemanager></angular-filemanager>

but still nothing happens,

Is it possible that you give us the source code of the demo app or just a step by step example

**Comment by hmartos** [comment]
I think your problem is that HTML directive is `<angular-file-manager></angular-file-manager>` instead of `<angular-filemanager></angular-filemanager>` like the README says.

**Comment by starspn** [comment]
Thanks hmartos you are a star atleast now i can see something, I need to figure out how to make the create folder and upload file works. I plan on using this on my angular app, my backend for the app is written in java

**Comment by hmartos** [comment]
Thank you starspn, now we are at the same point!.
This is an amazing project, but it is not very well documented, so you have to carefully read the source files. anyway, the directive is the starting point so this should be corrected in README

**Comment by starspn** [comment]
Yes abosulety. it is an amazing project, I wish it was well documented or atleast they give us access to the source code of the demo

**Comment by joni2back** [comment]
Hello, yes as @hmartos says, the directive was named from ```angular-file-manager``` to ```angular-filemanager``` in the latest release.
Now you need to extend the config file ... look the [index.html](https://github.com/joni2back/angular-filemanager/blob/master/index.html) main file (the same that includes the directive) in order to set your backend urls and enable/disable features.
The options are: 
```
listUrl: "bridges/php/handler.php",
uploadUrl: "bridges/php/handler.php",
renameUrl: "bridges/php/handler.php",
copyUrl: "bridges/php/handler.php",
removeUrl: "bridges/php/handler.php",
editUrl: "bridges/php/handler.php",
getContentUrl: "bridges/php/handler.php",
createFolderUrl: "bridges/php/handler.php",
downloadFileUrl: "bridges/php/handler.php",
compressUrl: "bridges/php/handler.php",
extractUrl: "bridges/php/handler.php",
permissionsUrl: "bridges/php/handler.php",
enablePermissionsRecursive: true,
sidebar: true,
breadcrumb: true,
allowedActions: {
    rename: true,
    copy: true,
    edit: true,
    changePermissions: true,
    compress: true,
    compressChooseName: true,
    extract: true,
    download: true,
    preview: true,
    remove: true
},
```

Then, backend should work in in the way that the [API](https://github.com/joni2back/angular-filemanager/blob/master/API.md) is documented.
Please if you still trying without success, please tell me.

If you can, you could modify the README.md docs then propose a pull request in order to clarify this issues.
Thanks!

**Comment by starspn** [comment]
Are these options for the Directive?

**Comment by hmartos** [comment]
Thank you @joni2back I will try to do a pull request tomorrow, but it is my first time, so I don't know how much time it will take me to do it!

I have overrrided this config file and it is working fine, but I am getting problems when trying to override this config file from fileManagerConfigProvider like you do in your index.html

  ```
    //example to override angular-filemanager default config
    angular.module('FileManagerApp').config(['fileManagerConfigProvider', function (fileManagerConfig) {
      var defaults = fileManagerConfig.$get();
      fileManagerConfig.set({
        appName: 'Demo - github.com/joni2back/angular-filemanager',
        allowedActions: angular.extend(defaults.allowedActions, {
          remove: true
        })
      });
    }]);
```

Thanks for your support!

**Comment by joni2back** [comment]
What problem?
If you are modifying the config.js file you should re-compile the javascript (run ```gulp build```)

**Comment by hmartos** [comment]
Sorry, the problem is `Error: [$injector:unpr] Unknown provider: fileManagerConfigProvider`.
I am importing source files directly in my html,  not using minified javascript, so I don't need to recompile the javascript, do I?

```
  <script src="/bower_components/angular-filemanager/src/js/app.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/config.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/chmod.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/item.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/filenavigator.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/fileuploader.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/translations.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/controller.js"></script>
  <script src="/bower_components/angular-filemanager/src/js/selector-controller.js"></script>
```

**Comment by joni2back** [comment]
Im trying the same and i dont have problems.. The source is up to date? (check config.js was transformed from constant to provider)

**Comment by hmartos** [comment]
Thank ypu @joni2back, that was the problem!!

**Comment by joni2back** [comment]
You're welcome. Dont forget to give the project a start :star:

**Comment by hmartos** [comment]
Starred! I think all this problems are because this version is not available on bower, am I wrong?
This code is tagged as 1.2.0 and it should be tagged as 1.3.1 ??

Thank you.

**Comment by joni2back** [comment]
Thank you again, I just released the ***1.3.1*** !

**Comment by starspn** [comment]
Thank you.. I will update my bower

On Tue, Aug 25, 2015 at 3:05 PM, Jonas Sciangula Street <
notifications@github.com> wrote:

> Thank you again, I just released the *1.3.1* !
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joni2back/angular-filemanager/issues/55#issuecomment-134578848>
> .
>



-- 
*Regards*


*Sibusiso Nkambulecell : 073 955 9087email : sbssnkambule@gmail.com
<sbssnkambule@gmail.com>*
